### ISSUE TITLE
qualifier within a field is not properly handled when it appears right before a newline

### ISSUE THREAD

**Issue by girishc** [Appendium/flatpack#22]
Here is an example. Let's suppose we have a file with | as delimiter and " as qualifier

This works:
Field1|Field2|"Field 3
is multiline
hence within "quotes" here."

But this fails:
Field1|Field2|"Field 3
is multiline
hence within "quotes"
here"

**Comment by benoitx** [comment]
Thanks for reporting this.  Could you create a unit test for this please?  Any suitable fix will also be gratefully accepted. Many thanks
Benoit

**Comment by benoitx** [comment]
Hi,
I assume that there is no fix for this in your version?  Any patch would be welcomed.

**Comment by benoitx** [comment]
Hi @martindiphoorn do you understand this report? I'm not sure I can see the test case girishc is trying to tell...

**Comment by martindiphoorn** [comment]
Is this ticket really 2 years old? Good change its fixed already. I have seen multiline handling in the code. But maybe its a new line directly after the field qualifier? So "<cr><nl>data" or maybe only a new line?

**Comment by benoitx** [comment]
Yep...2y old but I didn't get a positive reply that it was fixed or any actual test case.

We should try to add a specific test as you described and see what happens.
Is it something you could take care of? I'm away from my computer this weekend and then it is meetings after meetings at the beginning of the week... Let me know, thanks.

**Comment by martindiphoorn** [comment]
Yes i will look into it. Can you assign it to me?

**Comment by benoitx** [comment]
Would love to assign it to you but I can select your name it seems...

**Comment by martindiphoorn** [comment]
Added some extra test cases, but could not find a way to break it.
We should probably close this.
